run-coverage

```diff
@@ -11,8 +11,8 @@ except ImportError:
     exit(1)"
 
 rm -rf .coverage*
-PATH=$PWD:$PATH $COVERAGE run -p --include=$PWD/**/* ./tests/run-unit
-PATH=$PWD:$PATH RUNNER="$COVERAGE run -p --include=$PWD/**/*" ./run-test-ci
+PATH=$PWD:$PATH $COVERAGE run -p --include=$PWD/ocropus-*,$PWD/**/* ./tests/run-unit
```

Comment:
That seems redundant, `$PWD/**/*` glob should include `$PWD/ocropus-*`

Reply:
Did you try it out?

Reply:
Yes, it works with just `$PWD/**/*` for me. Does it not for you?

Reply:
I can't remember but I believe you. Feel free to change it then. A merge here is overdue.